CHANGES.md

```diff
@@ -1,3 +1,28 @@
+# snare 0.4.0 (2020-xx-xx)
+
+## Breaking changes
+
+* The `email` option in `match` blocks has been replaced by the more generic
+  `errorcmd`. To obtain the previous behaviour:
+
+    ```
+    email = "someone@example.com";
+    ```
+
+  should be changed to:
+
+    ```
+    errorcmd = "cat %s | mailx -s \"snare error: github.com/%o/%r\" someone@example.com";
+    ```
+
+  As this example may suggest, `errorcmd` is much more flexible than `email`.
+  The syntax of `errorcmd` is the same as `cmd` with the addition that `%s` is
+  expanded to the path of the failed job's combined stderr / stdout.
```

Comment:
This is what I was confused about before.

So since snare writes stdout/err to disk, you would never want your `errorcmd` to take input from a pipe anyway (unless you changed snare).

Reply:
I don't understand what you mean.

Reply:
In the snare_scripts repo I was questioning where the file came from, and this explains.

No action necessary on your part.

Reply:
OK.